quic/s2n-quic-core/src/stateless_reset/token.rs

```diff
@@ -18,7 +18,7 @@ pub const LEN: usize = 128 / 8;
 // The implemented PartialEq will have the same results as
 // a derived version, except it is constant-time. Therefore
 // Hash can still be derived.
-#[allow(clippy::derive_hash_xor_eq)]
```

Comment:
What if you did something like:

#[allow(unknown_lints, clippy::derive_hash_xor_eq, clippy::derived_hash_with_manual_eq)]

Reply:
Just tested it and that does allow us to continue using clippy 1.63. It's not super pretty, but probably better than making backwards incompatible clippy changes?

Reply:
Actually, I'm not sure what I missed when testing locally, but clippy still warns about the old name, (because it _does_ recognize it) and then we error out on that.

Reply:
seems that trick used to work: https://github.com/aws/s2n-quic/blob/aec1ff9bf46f05065edfc1c2766ad95c90858ef2/common/s2n-codec/src/decoder/value.rs#L104-L109
[3:55](https://amzn-aws.slack.com/archives/D047T5Z3DLY/p1679439345463349)
but they deprecated clippy::unknown_clippy_lints so I guess no longer